bot/exts/moderation/infraction/_utils.py

```diff
@@ -280,3 +283,25 @@ async def send_private_embed(user: MemberOrUser, embed: discord.Embed) -> bool:
             "The user either could not be retrieved or probably disabled their DMs."
         )
         return False
+
+
+class StaffBanConfirmationView(interactions.ViewWithUserAndRoleCheck):
+    """The confirmation view that is sent when a moderator attempts to ban a staff member."""
+
+    @discord.ui.button(label="Confirm", style=ButtonStyle.red)
+    async def confirm(self, interaction: Interaction, button: Button) -> None:
+        """Callback coroutine that is called when the "confirm" button is pressed."""
+        await interaction.response.send_message("Applied infraction.")
```

Comment:
I don't think we need this message, since feedback is managed by the ban command itself.

![image](https://github.com/python-discord/bot/assets/9753350/1da60176-8bf1-4000-b238-0ca73a6865c9)

Reply:
Should I also remove the response that says "Cancelled infraction" if the invoker hits the "Cancel" button?

Reply:
Nah, the cancel makes sense as it is, it was just eh double feedback message from the confirm that looks strange to me.